src/client/src/pages/DataView360/View/View360.js

```diff
@@ -70,8 +70,11 @@ class View360 extends Component {
             });
         response = await response.json();
 
-        let animalInfo = await fetch(`/api/person/${this.state.matchId}/animals`);
-        animalInfo = await animalInfo.json()
+        let shelterluvInfo = await fetch(`/api/person/${this.state.matchId}/animals`);
+        shelterluvInfo = await shelterluvInfo.json()
+        const shelterluvShortId = shelterluvInfo["person_details"]["shelterluv_short_id"]
```

Comment:
better to use `_.get(shelterluvInfo, "person_details.shelterluv_short_id")`
This way, In case person_details in undefined, the page won't crash